.github/workflows/srtool.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - master
+    tags:
+      - "[0-9].*"
```

Comment:
does it mean that this workflow will be triggered on a tag creation?

Reply:
Yes that is correct

Reply:
We have used tags of the 2.0.30 format so I continued that